packages/dashboard/src/project/deleteProject.tsx

```diff
@@ -0,0 +1,113 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import WarningIcon from '@mui/icons-material/Warning';
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Typography,
+} from '@mui/material';
+import { Paper } from '@sorry-cypress/dashboard/components';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router';
+import { useDeleteProjectMutation } from '../generated/graphql';
+import { useCurrentProjectId } from './hook/useCurrentProjectId';
+
+export const DeleteProject = () => {
+  const history = useHistory();
+  const projectId = useCurrentProjectId();
+  const [deleting, setDeleting] = useState(false);
+  const [deleteError, setDeleteError] = useState<string | null>(null);
+  const [shouldShowModal, setShowModal] = useState(false);
+
+  const [startDeleteProjectMutation] = useDeleteProjectMutation({
+    variables: {
+      projectId,
+    },
+  });
+
+  function deleteProject() {
+    setDeleting(true);
+    startDeleteProjectMutation()
+      .then((result) => {
+        if (result.errors) {
+          setDeleteError(result.errors[0].message);
+          setDeleting(false);
+        } else {
+          setDeleting(false);
+          setShowModal(false);
+        }
+        history.push('/');
+      })
+      .catch((error) => {
+        setDeleting(false);
+        setDeleteError(error.toString());
+      });
+  }
+
+  return (
+    <>
+      <Paper>
+        <Typography variant="h6">Remove Project</Typography>
+        <Typography variant="body1" gutterBottom component="div">
+          By removing this project, all runs will be deleted and will no longer
+          be available.
+        </Typography>
+        <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
+          <Button
+            variant="contained"
+            color="error"
+            onClick={() => setShowModal(true)}
+            disabled={deleting}
+          >
+            <DeleteIcon />
+            <Typography variant="body1">
+              {deleting ? 'Deleting' : 'Delete'}
+            </Typography>
+          </Button>
```

Comment:
```suggestion
          <Button
            variant="contained"
            color="error"
            onClick={() => setShowModal(true)}
            startIcon={<DeleteIcon />
            disabled={deleting}
          >
              {deleting ? 'Deleting' : 'Delete'}
          </Button>
```

![2021-11-07 at 6 02 PM](https://user-images.githubusercontent.com/5295554/140673144-c5b46265-1a3f-41a6-9d96-a2581979cfe8.jpg)

This way icon and text will be aligned.
You don't need Typography. all icons should have same typography applied by the button itself.

Reply:
done